core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -467,14 +467,15 @@ class CityInfo {
     }
 
     fun matchesFilter(filter: String): Boolean {
-        return when {
-            filter == "in this city" -> true
-            filter == "in all cities" -> true
-            filter == "in all coastal cities" && getCenterTile().isCoastalTile() -> true
-            filter == "in capital" && isCapital() -> true
-            filter == "in all cities with a world wonder" && cityConstructions.getBuiltBuildings().any { it.isWonder } -> true
-            filter == "in all cities connected to capital" -> isConnectedToCapital()
-            filter == "in all cities with a garrison" && getCenterTile().militaryUnit != null -> true
+        return when (filter) {
+            "in this city" -> true
+            "in all cities" -> true
+            "in all coastal cities" -> getCenterTile().isCoastalTile()
+            "in capital" -> isCapital()
+            "in all non-occupied cities" -> !cityStats.hasExtraAnnexUnhappiness()
```

Comment:
Do you mean puppeted? "in all non-puppeted cities"?

Reply:
This is the official wording in the game, but it wasn't clear to me what exactly it meant either. On internet I found a [reddit](https://www.reddit.com/r/civ/comments/5oq3c1/civ5_meritocracy_unhappiness_bonus_do_annexed/) post saying that this effect applies to all cities that are annexed but don't have a courthouse yet. I don't know whether this applies to puppetted cities as well.

Reply:
For now I'll assume it to mean 'annexed but without courthouse || puppetted'.

Reply:
Ok
We can always change it later